### ISSUE TITLE
Fix soloUMIposition documentation

### ISSUE THREAD

**Issue by ghuls** [alexdobin/STAR#789]
The example shows `soloCBposition` instead of `soloUMIposition`:
```
--soloUMIposition
default: -
string position of the UMI on the barcode read, same as soloCBposition
Example: inDrop (Zilionis et al, Nat. Protocols, 2017):
–-soloCBposition 3 9 3 14
```

**Comment by ghuls** [comment]
```
TopCells
only report top cells by UMI count, followed by the excat number of
cells
```
*ex**ac**t*

**Comment by alexdobin** [comment]
Thanks - fixed in upcoming 2.7.4a
